### PR TITLE
Add Britt Daniel quote to ch5

### PR DESCRIPTION
**Pull request by jni**

### CODE REVIEW

File: markdown/ch5.markdown [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Contingency tables using sparse coordinate matrices
 
+> I like sparseness. There's something about that minimalist feel that can make
+> something have an immediate impact and make it unique.
+>
+> — Britt Daniel
```
Comment (jni) [inline markdown/ch5.markdown]: I thought about it, but we don't have any for the others... What do you think about "lead singer for Spoon"?

**Comment by stefanv** [comment]
How much do you like this quote?  I wonder if it's a bit of a stretch.

**Comment by jni** [comment]
A bit of a stretch, but only a bit. I like the "immediate impact" — because you're not spending all this time multiplying zeros! =P 

Also, did you miss the bit about the guitarist being Greg Wilson? And that they're from Austin? I took that as a sign that this should be the quote. =D

**Comment by stefanv** [comment]
I guess the "make it unique" part didn't fit very well in the context.  We could always replace that with "..." :)

**Comment by stefanv** [comment]
And, yes, the Greg Wilson mention did not go by unnoticed 🎸

**Comment by hdashnow** [comment]
I'm ambivalent. No one is going to get the Greg Wilson reference but us. I like the quote, but if anyone thinks of a better one I'm happy to consider others. That said, happy to go with it for the time being.

**Comment by jni** [comment]
How about I add the [...] and optionally the rest of the quote?

**Comment by jni** [comment]
@stefanv I still need to try out Rocksmith... =D

**Comment by hdashnow** [comment]
I'm actually pretty happy with the quote as is. It's clear now that it says it's a musician that he's referring to to sparseness in the context of music. But I think even if it doesn't fit our point perfectly, it alludes nicely to elegance.

**Pull request merged**